Bridges/HttpKernel.php

```diff
@@ -100,6 +100,10 @@ public function onRequest(ReactRequest $request, HttpResponse $response)
         if ($this->application instanceof TerminableInterface) {
             $this->application->terminate($syRequest, $syResponse);
         }
+        
+        if (is_a($this->application, '\Illuminate\Contracts\Http\Kernel')) {
+            $this->application->terminate($syRequest, $syResponse);
+        }
```

Comment:
Is there a reason why you don't go with `$this->application instanceof \Illuminate\Contracts\Http\Kernel`?